recipe/meta.yaml

```
@@ -18,26 +19,27 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python >={{ python_min }}
```

Following the docs, this should be 
```suggestion
    - python {{ python_min }}
```

---

Thank you! See

https://github.com/conda-forge/mne-bids-feedstock/commit/dba1b2c0d2383e861184a5ebda1dde6ea57b8955